ndio/remote/neurodata.py

```diff
@@ -208,24 +208,29 @@ def get_public_datasets_and_tokens(self):
                 datasets[dataset] = [t]
         return datasets
 
-    def getURL(self, url, token=''):
+    def getURL(self, url):
```

Comment:
These changes look good to me! Since Resources API is now pulled in, lets add Authentication checks to delUrl and postUrl (which would be mostly copy paste from getUrl).